### ISSUE TITLE
Upgrade to 2.0.0 from F-Droid, accounts gone

### ISSUE THREAD

**Issue by blizzz** [nextcloud/android#1691]
### Actual behaviour

I have had the 2.0.0 RC installed from F-Droid.  Today, I upgraded to the final. When launching the app, and going through the 2.0-Intro-screens, I did not end up with my accounts but with a blank app, asking me to configure one.

### Expected behaviour

My old, configured  accounts (2) should still have been there.
 
### Steps to reproduce
1. Have a RC from F-Droid installed (iirc the latest, same app, not -dev, https://f-droid.org/packages/com.nextcloud.client/)
2. Upgrade to final
3. Launch NC app

### Environment data
Android version: 4.4.4

Device model: Jolla C

Stock or customized system: Stock (Alien Dalvik)

Nextcloud app version: 2.0.0

Nextcloud server version: 12.0.3

**Comment by mario** [comment]
Beta/dev and finals are different apps, so this is expected.

**Comment by blizzz** [comment]
@mario nope,  it was the  Beta Version of Nextcloud app, not Nextcloud-beta

**Comment by blizzz** [comment]
Okay, they were Release Candidates, not Betas. Sorry for the  confusion. Adjusting description.

**Comment by mario** [comment]
Still, they were from Nextcloud Dev channel of F-Droid, were they not?

**Comment by blizzz** [comment]
@mario no, they weren't. Version 2.0.0RC9 of com.nextcloud.client it was.

**Comment by mario** [comment]
Strange then :( sorry then :-/

Thx for the report

**Comment by blizzz** [comment]
@mario where is the config supposed to be stored?

**Comment by mario** [comment]
Account manager. Might be related to one PR by Tobi, will have to check...

**Comment by blizzz** [comment]
um… and on file system level?

**Comment by mario** [comment]
Well, there's a database in the internal data folder of the app, but accounts are not stored there :)

@AndyScherzinger could you please check if one of the latests PRs by Tobi (that handled account removal) could have done something like this?

**Comment by mario** [comment]
Hm this wasn't even merged yet, but I think @tobiasKaminsky put it into the 2.0 release if I  remember right...

https://github.com/nextcloud/android/pull/1676

**Comment by blizzz** [comment]
> I have already created a 2.0.0 gplay release with this fix.

So, perhaps he didn't add it to F-droid? 😭 

```
$ ls -a nextcloud/          
.  ..  arthur@server1.com  arthur@server2.org  tmp
```

If the data folder should be there, it is not anymore.

**Comment by mario** [comment]
Entirely possible since it wasn't merged, it's not in F-Droid, indeed :-/

@AndyScherzinger can we merge (even if it's late) and tag 2.0.1? *shrug* Doesn't have to be in Gplay at all, just for F-droid...

**Comment by jancborchardt** [comment]
Can confirm this after update via F-Droid of the regular app. Needed to authenticate again, but the files are still there locally (used SD card as storage).

@ogasser also mentioned this to me so maybe he has more info?

**Comment by blizzz** [comment]
@jancborchardt the cause is an unmerged fix that went into the Google store manually, but not to f-droid.

**Comment by AndyScherzinger** [comment]
@mario sure, go ahead. I won’t be near a keyboard for the next couple of hours...

**Comment by AndyScherzinger** [comment]
@mario I created a hotfix branch + the changes #1696

**Comment by AndyScherzinger** [comment]
solved via #1696 which should get picked up by f-droid tomorrow or the day after (hopefully).
